components/proxy/src/main/java/com/hotels/styx/routing/config/Builtins.java

```diff
@@ -87,35 +97,42 @@
     private Builtins() {
     }
 
-    public static RoutingObject build(List<String> parents, RoutingObjectFactory.Context context, RoutingObjectConfiguration configNode) {
-        if (configNode instanceof RoutingObjectDefinition) {
-            RoutingObjectDefinition configBlock = (RoutingObjectDefinition) configNode;
+    public static RoutingObject build(List<String> parents, RoutingObjectFactory.Context context, StyxObjectConfiguration configNode) {
```

Comment:
We have three public methods here named `build` but with different arguments. I think some comments/documentation are in order here to explain what they are for and the differences between them.

Reply:
Okay. The return type gives a clue what they are for. But will add documentation regardless.